packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -85,6 +85,10 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
 
         this._isDeinitialize = false;
 
+        this._loadStripeElement(stripeupe, gatewayId, methodId).catch((error) =>
```

Comment:
Can we add/update some tests to test this behaviour?

Reply:
I've already added the on mock to cover functionality